ruby/.github/workflows/oop-milestone-1.yml

```diff
@@ -0,0 +1,20 @@
+name: Tests
+
+on: pull_request
+
+jobs:
+  milestone-1:
+    name: Project Setup
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
+      - name: Setup RSpec
+        run: |
+          gem install --no-document rspec:'~>3.0'
+          rspec --init
+          wget -P spec/ https://raw.githubusercontent.com/microverseinc/linters-config/gh-actions/ruby/ruby/oop/spec/milestone_1_spec.rb
```

Comment:
This URL has to be updated before merging to point to the master branch: `https://raw.githubusercontent.com/microverseinc/linters-config/master/ruby/oop/spec/milestone_1_spec.rb`